content/getting-started/readfiles/framework-deploy-note.md

```diff
@@ -0,0 +1,8 @@
+Framework Deploy AKA [framework-deploy](https://github.com/cisco-sso/framework-deploy) is a boilerplate git repo in
+the KDF deployment pattern. The boilerplate structure in framework-deploy is made to fit perfectly with the tools
+available in the KDK container. 
+
+Create a private fork of platform-deploy and use it to hold non-secret Kubernetes-related application configuration
```

Comment:
Are we calling it framework-deploy or platform-deploy?

Reply:
My mistake. Fixed it. Good catch!  Thanks!